src/html/header.php

```diff
@@ -4,10 +4,21 @@
             <nav class="navbar navbar-default">
                 <div class="container-fluid">
                     <div class="navbar-header">
+                        <!-- Burger menu -->
+                        <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" id="bs-example-navbar-collapse-1 aria-expanded="false">
+                            <span class="sr-only">Toggle navigation</span>
+                            <span class="icon-bar"></span>
+                            <span class="icon-bar"></span>
+                            <span class="icon-bar"></span>
+                        </button>
+                        <!-- End burger menu -->
+                        <!-- Logo -->
                         <a class="navbar-brand" href="#">
                             <img src="http://via.placeholder.com/70x70" alt="logo">
                         </a>
                     </div>
+                        <!-- End logo -->
+                    
                     <ul class="nav navbar-nav navbar-right">
```

Comment:
before the 
```html
<ul class="nav navbar-nav navbar-right">
```
can you add a div tag : 
```html
<div class="collapse navbar-collapse" id="top-navbar">
```

and close the tag, between the lines 26 and 27, with a close </div> tag